sdk/core/System.ClientModel/src/Message/BinaryContent.cs

```diff
@@ -125,6 +167,41 @@ public override async Task WriteToAsync(Stream stream, CancellationToken cancell
         public override void Dispose() { }
     }
 
+    private sealed class JsonBinaryContent : BinaryContent
```

Comment:
remove this. it's not needed. just return BinaryContent.Crete(BinaryData.FromObjectAsJson(...));